payroll/views.py

```diff
@@ -240,9 +230,14 @@ def composition_data(self):
         composition_json = []
         percentage_tracker = 0
 
+        unit_base_pay, unit_extra_pay = super().get_entity_payroll()
```

Comment:
You can call this method via `self`. I generally use `super()` if I need to extend a function from a parent class, instead of overriding it, e.g., when we define `get_context_data` on the `Unit` and `Department` view classes, we call `super().get_context_data()` first to run the method from the parent `EmployerView`, then we do additional things based on the unit or department. [Here's more on `super()`](https://realpython.com/python-super/).